packages/iov-ethereum/src/testconfig.ts

```diff
@@ -35,6 +41,12 @@ const local: EthereumNetworkConfig = {
   nonce: new Int53(0) as Nonce,
   gasPrice: "20000000000",
   gasLimit: "2100000",
+  waitForTx: 100,
+  scraper: {
+    api: "https://api-ropsten.etherscan.io/api",
+    chainId: "3" as ChainId,
+    address: "0x0A65766695A712Af41B5cfECAaD217B1a11CB22A" as Address,
+  },
```

Comment:
Is this supposed to be here? How can https://api-ropsten.etherscan.io/api index the local chain?

Reply:
I made this way (ugly) to allow test in CI, but forgot to `export ETHEREUM_SCRAPER=1` to travis.sh, should I add the export value or skip this test in travis?

Reply:
Since there is no scaper for the clocal chain, I would just skip it and set `scraper: undefined` in the config. scraper+blockchain should belong together. Otherwise it gets very confusing.

We can test other networks manually on dev machines and later add a scraper for the local chain.